content/docs/development/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Development"
```

Comment:
At some point we may want to put a list of the development pages on this page. We could probably write a partial to build the list for us automatically.

Reply:
Sounds good to me.